pandas/tests/series/indexing/test_indexing.py

```diff
@@ -753,6 +753,16 @@ def test_take():
         s.take([-1, 3, 4], convert=False)
 
 
+def test_take_categorical():
```

Comment:
instead of this test (or in addition to), I think you should be able to remove the skip from `test_take_series` in the categorical extension tests

Reply:
That still fails because of https://github.com/pandas-dev/pandas/issues/20747. The unobserved categories are still present. I've updated the skip.

Reply:
Ah, yes, that's still there :)